frontend/src/main/resources/application.properties

```diff
@@ -0,0 +1,3 @@
+spring.h2.console.enabled=true
```

Comment:
Why is this here? There is no database atm

Reply:
i was in the process of adding one, should i take i out for now

Reply:
yes